.github/workflows/staging-workflow.yml

```diff
@@ -0,0 +1,48 @@
+name: Testing the translated Documentation
+
+on:
+  pull_request:
+    branches: [ master ]
+
+jobs:
+  build:
+    runs-on: ubuntu-latest
+    strategy:
+      matrix:
+        language: [de_DE, ja_JP, es_ES, es_MX, fr_FR, it_IT, ko_KR, pt_BR, pt_PT, tr_TR]
+
+    # if: "contains(github.event.head_commit.message, 'New translations')"
+    steps:
+    - uses: actions/checkout@v2
+
+    - name: Set up Python
+      uses: actions/setup-python@v2
+      with:
+        python-version: '3.8'
+
+    - name: Install dependencies
+      run: |
+        python -m pip install --upgrade pip
+        pip install -U virtualenv setuptools wheel
+        pip install -U -r requirements-dev.txt sphinx-intl qiskit
+        sudo apt-get install graphviz pandoc
+
+    - name: Build translated Qiskit Docs
+      env:
+        TRANSLATION_LANG: ${{ matrix.language }}
+      run: |
+        tools/staging_translated_documentation.sh
+
+    - name: Install SSH Client
+      uses: webfactory/ssh-agent@v0.2.0
+      with:
+          ssh-private-key: ${{ secrets.GH_PAGES_DEPLOY_KEY }}
```

Comment:
This won't work with pull requests from a fork. I think that's fine here for the expected use case here which is to update the staging site with new updates from crowdin. But it's just something to be aware of. It might also cause a red x on the checks list for PRs that are from a a fork.

Reply:
Yes, I understood this from your [PR in terra](https://github.com/Qiskit/qiskit-terra/commit/2c5a09cabe0da0bf548ed11cd101ce501375c896). Thank you for pointing that out. Also regarding the landing page, I just added [index.html](https://github.com/qiskit-community/qiskit-translations/blob/gh-pages/index.html).

Reply:
I'm just a little worried about the build failing for random languages sometimes.